cmd/containerd-nydus-grpc/pkg/command/flags.go

```diff
@@ -182,6 +183,11 @@ func buildFlags(args *Args) []cli.Flag {
 			Usage:       "whether to disable nydus-overlayfs to mount",
 			Destination: &args.EnableNydusOverlayFS,
 		},
+		&cli.IntFlag{
+			Name:        "nydusd-thread-num",
+			Usage:       "Nydusd daemon thread-num, default will be set to the number of CPUs",
```

Comment:
I am not very sure if make the default threads number to the number of CPUs is a suitable policy.
Consider that a server equipped with many CPU sockets and cores, dozens of threads will be spawned. But most of them are idle.

Reply:
It seems nydusd is using a default of `1`, so if we don't want to set a default value from snapsohtter, we can just don't pass this argument to nydusd if the user doesn't set it.

Reply:
Sounds reasonable

Reply:
fixed.